.github/workflows/build_packages.yaml

```diff
@@ -557,6 +606,78 @@ jobs:
         aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
         docker push public.ecr.aws/emqx/emqx:${version#v}
 
+  docker-push-multi-arch-manifest:
+    # note, we only run on amd64
+    if: >
+      (github.event_name == 'release' && !github.event.release.prerelease)
+      || (github.event.repository.owner != 'emqx' && startsWith(github.ref_name, 'ci/'))
+    needs:
+      - prepare
+      - docker
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        profile: # all editions for docker
+          - emqx-edge
+          - emqx
+          - emqx-enterprise
+        arch:
+          - amd64
+          - arm64
+        build_machine:
+          - aws-arm64
+          - ubuntu-20.04
+        exclude:
+        - arch: arm64
+          build_machine: ubuntu-20.04
+        - arch: amd64
+          build_machine: aws-arm64
+        # NOTE: for docker, only support latest otp version, not a matrix
+        otp:
+          - 24.1.5-3 # update to latest
+
+    steps:
+      - uses: actions/download-artifact@v2
+        if: matrix.arch == 'amd64'
+        with:
+          name: source
+          path: .
+
+      - name: unzip source code
+        if:  matrix.arch == 'amd64'
+        run: unzip -q source.zip
+
+      - uses: docker/login-action@v1
+        if: matrix.arch == 'amd64'
+        with:
+          username: ${{ secrets.DOCKER_HUB_USER }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+      # NOTE, Pls make sure this is identical as the one in job 'docker'
+      - uses: docker/metadata-action@v3
+        if: matrix.arch == 'amd64'
+        id: meta
+        with:
+          images: ${{ github.repository_owner }}/${{ matrix.profile }}
+          flavor: |
+            latest=false
+            suffix=-${{ matrix.arch }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=ref,event=tag
+            type=semver,pattern={{version}}
+          labels:
+            org.opencontainers.image.otp.version=${{ matrix.otp }}
```

Comment:
Q: will this job get the `-elixir` tagged image as well?

Reply:
No. I see elixir build is only for x86_64, there is no arm64 so this job is not necessary for elixir or?

Reply:
All these [steps](https://github.com/emqx/emqx/pull/6880/files/f4724fde0df1d694fb538d0705b8474a0fa6d5e4#diff-d2796f5d4197787c17ee689a8fc9bed7cb4d73cea4af047c47028e6a5e4f5f9aR659) are checking for `amd64`, so it would include Elixir.

Maybe the intention was to check for `arm64` then?

Reply:
No. the intention is to get `steps` run on amd64 host only (not arm64) since the tasks are just calling docker commands to update manifests it is arch independent and no need to get executed twice.